src/main/java/com/spotify/futures/CompletableFuturesExtra.java

```diff
@@ -55,4 +60,84 @@ private CompletableFuturesExtra() {
     }
     return new CompletableToListenableFutureWrapper<V>(future);
   }
+
+  public static <T> CompletableFuture<T> immediatelyFailed(Throwable throwable) {
+    final CompletableFuture<T> future = new CompletableFuture<T>();
+    future.completeExceptionally(throwable);
+    return future;
+  }
+
+  public static <T> CompletableFuture<T> exceptionallyCompose(
```

Comment:
Also, re the name, I think 'exceptionally' is better; it's the same name that is used in CompletionStage for the method that does 'the same thing'. It's not so much composition as it is a fallback, I think?